### PR TITLE
perf(spec-parser): support post and update server validation

### PR DESCRIPTION
**Pull request by SLdragon**
1. Support post operation
2. Update server validation

### CODE REVIEW

File: packages/fx-core/src/common/spec-parser/manifestUpdater.ts [hunk 1]
```diff
@@ -55,40 +88,51 @@ export async function generateCommands(
...
+                  parameters.push({
+                    name: param.name,
+                    title: updateFirstLetter(param.name),
+                    description: param.description ?? "",
+                  });
+                });
+              }
+
+              if (operationItem.requestBody) {
+                const requestJson = (operationItem.requestBody as OpenAPIV3.RequestBodyObject)
```
Comment (yuqizhou77) [inline packages/fx-core/src/common/spec-parser/manifestUpdater.ts:110]: Do we check content to be "application/json" when validating? or any requirements about content type? Just trying to think about possible values when we run generate()

Comment (SLdragon) [reply packages/fx-core/src/common/spec-parser/manifestUpdater.ts:110]: From Josh's validation rule, the request body should be "application/json", other type is not supported

Comment (yuqizhou77) [reply packages/fx-core/src/common/spec-parser/manifestUpdater.ts:110]: so we will have plan to update our validate() then?

**Review by yuqizhou77 — DISMISSED** [review]

File: packages/fx-core/src/common/spec-parser/manifestUpdater.ts (hunk 1 above)
Comment (yuqizhou77) [reply packages/fx-core/src/common/spec-parser/manifestUpdater.ts:110]: noticed that we've already have this in this PR. Resolved.

**Review by yuqizhou77 — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**